httpbakery/agent/agent.go

```diff
@@ -78,11 +70,38 @@ type agentLogin struct {
 	PublicKey *bakery.PublicKey `json:"public_key"`
 }
 
-// agentResponse defines the structure of an agent login response.
+// agentResponse contains the response to an agent login attempt.
 type agentResponse struct {
 	AgentLogin bool `json:"agent_login"`
 }
 
+var ErrNoAgentLoginCookie = errgo.New("no agent login cookie found")
```

Comment:
A doc comment would be nice. And possibly it should be spelled "agent-login" (not sure on that one)